web/components/App.jsx

```diff
@@ -58,8 +57,10 @@ class App extends React.Component{
       setTimeout(this.closeToast, TOAST_TIMEOUT);
     }
     if (prevProps.recommendations.length !== this.props.recommendations.length) {
+      const recommendationsToSet = this.props.recommendations.map(({ label }) => label);
+      // eslint-disable-next-line no-unused-expressions
       this.pageInstrumentBuilder
```

Comment:
Why this guy sometimes is undefined?

Reply:
I believe it's a condition where it hasn't been initialised yet